app/views/components/input_textarea.scala.html

```diff
@@ -26,27 +26,31 @@
         maxlength: Int = 90000
 )(implicit messages: Messages)
 
+@ariaDescribeElements={@hint.map{_ =>@{field.id}-hint}@field.error.map{_ => error-message-@{field.id}-input}}
+
 <div class="form-group @if(field.hasErrors){form-group-error}">
-    <label class="form-label" for="@{field.id}">
-        @labelHtml.map {lb => @lb}
-        @label.map { label =>
-            <div class="@labelClass">@label</div>
-        }
-        @hint.map { hint =>
-            <div class="form-hint">@hint</div>
-        }
-        @field.errors.map { error =>
-            <span class="error-message" id="error-message-@{field.id}-input">@messages(error.message, error.args: _*)</span>
-        }
-    </label>
+
+    @label.map { label =>
+        <label class="form-label @labelClass" for="@{field.id}">@label</label>
+    }
+    @labelHtml.map { label =>
+        <label for="@{field.id}">@label</label>
```

Comment:
Is this not going to result in two <labels>?  Before labelHtml was the content for a label; now it's a whole other <label>.  Or am I missing something?